core/domain/question_services.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Services for questions data model."""
+
+from core.platform import models
+
+import feconf
+
+(question_models,) = models.Registry.import_models([models.NAMES.question])
+
+# This takes additional 'title' parameters.
+CMD_CREATE_NEW = 'create_new'
+
+def _create_question(committer_id, question, commit_message, commit_cmds):
+    """Creates a new question.
+
+    Args:
+        committer_id: str. ID of the committer.
+        question: Question. question domain object.
+        commit_message: str. A description of changes made to the question.
+        commit_cmds: list(dict). A list of change commands made to the given
```

Comment:
I think we should start having a domain object to represent these commit_cmds -- perhaps QuestionChangeList or similar.

Reply:
I will add it in the next milestone?

Reply:
Hm, no, don't commit stuff that you'll know you'll have to change later. I suggest doing it properly right from the get-go, or leaving it out altogether from the PR.

Reply:
Removed commit_cmds. Will update this after implementation of QuestionChangeList domain object. Added a TODO comment for the same.